R/server.R

```diff
@@ -110,12 +110,22 @@ server <- function(input, output, session) {
     if (!exists("temp_md_scenario")) temp_md_scenario <- tempfile(fileext = ".md")
     file_conn <- file(temp_md_scenario)
     scenario_no <- c(
-      which(sapply(scenarios, `[[`, i = "name") == report_selection),
+      which(sapply(global$scenarios, `[[`, i = "name") == report_selection),
       length(report_contents()) - 1
     )
+    tempfun=function(x) gsub("\\(([[:graph:]]*)(.png)", paste0("(", getwd(),"/www/", "\\1\\2"), x,perl=T)
```

Comment:
better name for function, also prefer <-

Reply:
addressed